test/Unit/Util/PhpstormHelperTest.php

```diff
@@ -94,5 +94,76 @@ public function providePhpstormCases(): Generator
             $paratestBinary,
             $phpunitBinary,
         ];
+
+        $argv   = [];
+        $argv[] = $paratestBinary;
+        $argv[] = $phpunitBinary;
+        $argv[] = '--runner';
+        $argv[] = 'WrapperRunner';
+        $argv[] = '-dxdebug.mode=coverage';
+        $argv[] = '--configuration';
+        $argv[] = '/home/user/repos/test/phpunit.xml';
+        $argv[] = '--teamcity';
+
+        $expected   = [];
+        $expected[] = $paratestBinary;
+        $expected[] = '--runner';
+        $expected[] = 'WrapperRunner';
+        $expected[] = '--configuration';
+        $expected[] = '/home/user/repos/test/phpunit.xml';
+        $expected[] = '--teamcity';
+
+        yield 'with -dxdebug.mode=coverage run ParaTest' => [
+            $argv,
+            $expected,
+            $paratestBinary,
+            $paratestBinary,
+        ];
+
+        $argv   = [];
+        $argv[] = $paratestBinary;
+        $argv[] = $phpunitBinary;
+        $argv[] = '-dxdebug.mode=coverage';
+        $argv[] = '--configuration';
+        $argv[] = '/home/user/repos/test/phpunit.xml';
+        $argv[] = '--teamcity';
+
+        $expected   = [];
+        $expected[] = $paratestBinary;
+        $expected[] = '--configuration';
+        $expected[] = '/home/user/repos/test/phpunit.xml';
+        $expected[] = '--teamcity';
+
+        yield 'with -dxdebug.mode=coverage and no wrapper run ParaTest' => [
+            $argv,
+            $expected,
+            $paratestBinary,
+            $paratestBinary,
+        ];
+
+        $argv   = [];
+        $argv[] = $paratestBinary;
+        $argv[] = $phpunitBinary;
+        $argv[] = '--runner';
+        $argv[] = 'WrapperRunner';
+        $argv[] = '-dpcov.enabled=1';
```

Comment:
This isn't the order I get in my PHPStorm:

```php
array (
  0 => './bin/paratest_for_phpstorm',
  1 => '-dpcov.enabled=1',
  2 => '/home/tessarotto/repos/paratest/vendor/phpunit/phpunit/phpunit',
  3 => '--runner',
  4 => 'WrapperRunner',
  5 => '--coverage-clover',
  6 => '/home/tessarotto/.cache/JetBrains/PhpStorm2022.2/coverage/paratest@ParaTest.xml',
  7 => '--configuration',
  8 => '/home/tessarotto/repos/paratest/phpunit.xml',
  9 => '--teamcity',
)
```

Can you add this to data sets?
At this point the original `unset($argv[1]);` is not enough, we need to cycle all the `$argv` var and remove the first entry that ends with `/phpunit`

Reply:
Ok, the test cases are updated to match, the order of arguments on my machine matches yours. I also switched to searching for the necessary argv keys to unset instead of hard-coding the array keys. Let me know if this is what you were aiming for :)